test/client/fluentd/receiver_test.go

```diff
@@ -0,0 +1,49 @@
+package fluentd_test
+
+import (
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-logging-operator/test"
+	"github.com/openshift/cluster-logging-operator/test/client"
+	. "github.com/openshift/cluster-logging-operator/test/client/fluentd"
+	. "github.com/openshift/cluster-logging-operator/test/matchers"
+	"github.com/openshift/cluster-logging-operator/test/object"
+)
+
+var _ = Describe("Receiver", func() {
```

Comment:
Isn't defer ginkgo.GinkgoRecover() required for a testcase that makes use of go routines?